Editor/UI/Components/DocumentationButton.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+namespace ReadyPlayerMe.Core.Editor
+{
+    public static class DocumentationButton
+    {
+        public static void Draw(string url)
+        {
+            if (GUILayout.Button("?", new GUIStyle(GUI.skin.button)
```

Comment:
we can move the gui style out and use is as a private var I think, not it gets created everytime the button is rendered.